src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -167,6 +173,99 @@ public void testPullRequestAnalysisWithNewIssues() {
   }
 
   @Test
+  public void testSortIssues() {
+    ArgumentCaptor<String> commentCaptor = forClass(String.class);
+
```

Comment:
I'm usually in the middle... I mock dependencies that have a behaviour and use a bean when it's just data with no logic whatsoever.